daffodil-test/src/test/resources/org/apache/daffodil/section17/calc_value_properties/computedLengthFields.tdml

```diff
@@ -0,0 +1,215 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="DFDLTutorialStylesheet.xsl"?>
```

Comment:
What is this stylesheet line doing in a TDML file?

Reply:
removed. I was thinking of making this a tutorial - TDML files have a feature where you can put tutorial elements in, and the stylesheet, and then if you rename the file to ".tdml.xml" extension, browsers will stylize it. 

Check out https://daffodil.apache.org/tutorials/bitorder.tutorial.tdml.xml for an example. 

But then I ran into this bug, and ended up just turning it into a test file.